requirements_docs.txt

```diff
@@ -1,8 +1,5 @@
 mike~=2.1
-mkdocs~=1.5; python_version<"3.8"
-mkdocs~=1.6; python_version>="3.8"
+mkdocs~=1.6; python_version>="3.9"
```

Comment:
We can skip the `python_version>=3.9` part to simplify the requirements file. Same for the other lines.